configs/common/cores/arm/HPI.py

```diff
@@ -1683,6 +1683,11 @@ class HPI_MMU(ArmMMU):
 class HPI_BTB(SimpleBTB):
     numEntries = 128
     tagBits = 18
+    associativity = 1
+    btbReplPolicy = LRURP()
+    btbIndexingPolicy = BTBIndexingPolicy(
+        size="384B", entry_size=3, assoc=associativity
```

Comment:
Curiosity: where do the config numbers come from? (I am not too familiar with BTBs)

Reply:
It is approximately 'numEntires * entry_size' so that indexing is as before. I will re-work this part once before merging the final changes.